ontology/maxo.md

```diff
@@ -48,5 +48,3 @@ tags:
 tracker: https://github.com/monarch-initiative/MAxO/issues
 activity_status: active
 ---
```

Comment:
@LCCarmody It looks like the free text description was removed. This would result in a sparse page for MAXO at https://obofoundry.org/ontology/maxo.html

Ideally, we want to have a detailed description about the ontology in the free text.

Reply:
Would moving the 'description' to the free text (I think you mean the box below) suffice?

Reply:
Copying over the description from the metadata section to the main body of the markdown is definitely a good first step. Even better if you can provide more information about the ontology and its use (like the type of concepts defined, upper ontologies used, its application, projects that use the ontology, etc.).

Reply:
There should not be any redundancy between the description and the main body of the text, as both are displayed side by side. I would prefer if we keep this issue focused on the license change and not add additional complexity by adding a new request on top.